webanno-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/login/LoginPage.java

```diff
@@ -75,13 +79,23 @@
     private @SpringBean(required = false) SessionRegistry sessionRegistry;
 
     private LoginForm form;
+    private final MultiLineLabel tooManyUsersLabel;
+    private final Button signInBtn;
+    
     
     public LoginPage()
     {
         setStatelessHint(true);
         setVersioned(false);
         
         add(form = new LoginForm("loginForm"));
+        signInBtn = new Button("signInBtn", new ResourceModel("signIn"));
+        tooManyUsersLabel = new MultiLineLabel("usersLabel", new ResourceModel("tooManyUsers"));
+        tooManyUsersLabel.setVisible(false);
```

Comment:
Instead of setting the visibility of a component via `setVisible`, better use a behavior, e.g. 

```
        label.add(LambdaBehavior.visibleWhen(() -> getModelObject().getPreferences().isRememberLayer()));
```